FairBid/tapjoy.plist.xml

```diff
@@ -28,6 +28,290 @@
 				<key>SKAdNetworkIdentifier</key>
 				<string>578prtvx9j.skadnetwork</string>
 			</dict>
+			<dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>hs6bdukanm.skadnetwork</string>
+			</dict> 
+			 <dict>
+				<key>SKAdNetworkIdentifier</key>
+				<string>22mmun2rn5.skadnetwork</string>
+			</dict>
+            <dict>  
```

Comment:
The indentation is incorrect. We should keep the indentation of the files according to xml standard.